ci/build.yml

```diff
@@ -118,7 +118,7 @@ suite-web-landing build stable:
     expire_in: 1 days
 
 .build_nix: &build_nix
-  script: # override build script to use nix-shell instead
+  script:  # override build script to use nix-shell instead
```

Comment:
space is neccessary? same below

Reply:
not really but yaml linter issues warning over this missing space https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/1615420866

Reply:
ah, cool